macros/edr/dbt_artifacts/upload_dbt_tests.sql

```diff
@@ -133,12 +141,88 @@
     {{ return(flatten_test_metadata_dict) }}
 {% endmacro %}
 
-{% macro get_test_type(test_path) %}
+{% macro get_test_type(test_path, test_namespace = none) %}
     {% set test_type = 'generic' %}
-    {%- if 'tests/generic' in test_path or 'macros/' in test_path -%}
+    {%- if test_namespace == 'dbt_expectations' -%}
+        {% set test_type = 'expectation' %}
+    {%- elif 'tests/generic' in test_path or 'macros/' in test_path -%}
         {% set test_type = 'generic' %}
     {%- elif 'tests/' in test_path -%}
         {% set test_type = 'singular' %}
     {%- endif -%}
     {{- return(test_type) -}}
-{%- endmacro -%}
\ No newline at end of file
+{%- endmacro -%}
+
+{% macro get_default_description(short_name, test_namespace = none) %}
```

Comment:
Not sure if this is the right place for this macro 🤔 
It seems OK because it is only used here but LMK if you think of a better place to put it!